brand/models/brand_suggestion.py

```diff
@@ -0,0 +1,21 @@
+from brand.models.brand import Brand
+from datetime import datetime
+from django.db import models
+
+
+class BrandSuggestion(Brand):
+    """
+        This is the extension of the Brand container with the additional parameters.
+    """
+
+    submitter_name = models.CharField(verbose_name="Name of the submitter", max_length=100, null=False,
+                                      blank=False, default='unnamed')
+
+    submitter_email = models.EmailField("Email of the submitter", max_length=100, blank=True)
+
+    submission_date = models.DateTimeField(default=datetime.now, blank=True)
+
+    brand_suggestion_id = models.BigAutoField(primary_key=True)
```

Comment:
This field shouldn't be necessary since Django handles primary keys on its own:

https://docs.djangoproject.com/en/4.2/topics/db/models/#automatic-primary-key-fields

Reply:
Thanks for your comments. Yes, I will implement API portion before merging.